js/content/community.js

```diff
@@ -2863,7 +2873,17 @@ let FriendsThatPlayPageClass = (function(){
         defaultNode.insertAdjacentElement("afterend", sorted);
         HTML.afterEnd(defaultNode, "<div style='clear: both'></div>");
 
+        let active = "default";
```

Comment:
Could you save this information with CSS classes?

Reply:
Does that matter?

Reply:
It's more idiomatic

Reply:
i.e. following certain conventions

Reply:
Well, in other sorting pages, this state is maintained in LocalStorage instead. However, for some reason here there was no sorting state remembering, thus I didn't add it either. Frankly, the entire friendsthatplay page sorting is pretty broken. It doesn't properly sort everything and also breaks sometimes if it cannot find a play time.